models/lfs_lock.go

```diff
@@ -0,0 +1,173 @@
+// Copyright 2014 The Gogs Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package models
+
+import (
+	api "code.gitea.io/sdk/gitea"
+	"fmt"
+	"strconv"
+	"time"
+)
+
+// LFSLock represents a git lfs lfock of repository.
+type LFSLock struct {
+	ID          int64       `xorm:"pk autoincr"`
+	RepoID      int64       `xorm:"INDEX"`
+	Repo        *Repository `xorm:"-"`
+	OwnerID     int64       `xorm:"INDEX"`
+	Owner       *User       `xorm:"-"`
+	Path        string
+	Created     time.Time `xorm:"-"`
+	CreatedUnix int64     `xorm:"INDEX"`
```

Comment:
Use `created` is enough, then line 28 -> 30 and line 36 could be removed.

Reply:
I base myself on other models like (https://github.com/go-gitea/gitea/blob/a8717e5e3ace4dd226547faae9c3a44616bbf6f0/models/issue_stopwatch.go#L24) , after review more of those.
I might be needed adding `xorm:"INDEX created"` on created (like https://github.com/go-gitea/gitea/blob/aa962deec037375a7cfae3b96094becdfe48e778/models/token.go#L23) but after adding that does I need https://github.com/go-gitea/gitea/blob/aa962deec037375a7cfae3b96094becdfe48e778/models/token.go#L32 ? 
I will totally follow you on xorm since you will know more about it and maybe (in another PR) we should clean other models with such pattern.